src/pint/__init__.py

```diff
@@ -105,3 +107,31 @@
             astropy.__version__
         )
     )
+
+# location of tim, par files
+def datadir():
+    """Location of the PINT data (par and tim files)
+
+    Returns
+    -------
+    str
+        Directory of PINT data files
+
+    """
+    return pkg_resources.resource_filename(__name__, "data/")
+
+
+def datafile(filename):
+    """Full path to the requested PINT data file
+
+    Parameters
+    ----------
+    filename : str
+
+    Returns
+    -------
+    str
+        Full path to the requested file
+
+    """
+    return os.path.join(datadir(), filename)
```

Comment:
Would this be better as `...resource_filename("data/" + filename)`? I know the `pkg_resources` docs make a big deal about how resource names aren't file names...

Reply:
Not sure that I fully grasp the difference, but sure.

Reply:
I don't think there should be a difference; it should all be moot. I don't think PINT can be installed from a ZIP file (but maybe?) so all of them probably just return the file name. But I'm trying to figure out how to use `pkg_resources` as close to the way it is intended as possible.